free-courses-en.md

```diff
@@ -210,6 +217,7 @@
 
 ### Ruby
 
+* [CodeSchool - Getting Started With Ruby on Rails](https://www.codeschool.com/learn/ruby)
```

Comment:
Nope, i don´t think. There´s a [page](https://www.codeschool.com/pricing) with pricing plans. So, i guess, it´s not completely free.